Documentation/HowToGuides/PassingAFloatValueFromAnInteractorToAnInteractable/README.md

```diff
@@ -0,0 +1,190 @@
+# Passing A Float From An Interactor To An Interactable
+
+> * Level: Intermediate
+>
+> * Reading Time: 10 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction 
+
+## Prerequisites
+
+* [Add the Tilia.Interactions.Interactable.Unity] prefab to the scene Hierarchy.
+
+## Let's Start
+
+### Step 1
+
+Disable `CameraRigs.SpatialSimulator` and enable `CameraRigs.UnityXR`
+
+![Disable Camera Rig](assets/images/DisableCameraRig.png)
+
+### Step 2
+
+Expand the `Tilia Input UnityInputManager` Package directory in the Unity Project window and then select `Packages -> Runtime -> Prefabs -> ControllerMappings`. For this example we are going to add the `Input.UnityInputManager.Oculus.TouchLeftController` prefab and the `Input.UnityInputManager.Oculus.TouchRightController`prefab.
+
+Drag and Drop those two items into the Unity Scene.
+
+![Expand Directory](assets/images/ExpandDirectory.png)
+
+### Step 3
+
+We need to update our Interactors to listen to the Grip Button on our controller to grab the object. Expand `CameraRigs.TrackedAlias` and find the Interactors in the `LeftControllerAlias` and `RightControllerAlias`.
+
+For the `LeftControllerAlias`, Expand `CameraRigs.TrackedAlias -> Aliases -> LeftControllerAlias and click on the `Interactions.Interactor` Gameobject.
+
+Expand the `Input.UnityInputManager.Oculus.TouchLeftController -> Input Actions -> LeftGrip` and drag and drop the `LeftGrip_Press` GameObject onto the `Grab Action` parameter on the `Interactor Facade` component of the `CameraRigs.TrackedAlias -> Aliases -> LeftControllerAlias -> Interactions.Interactor` GameObject. 
+
+![Drag And Drop Left Grip Into Grab Action](assets/images/DragAndDropLeftGripIntoGrabAction.png)
```

Comment:
The arrow needs to go right down to `LeftGrip_Press`

![image](https://user-images.githubusercontent.com/1029673/74438181-821db280-4e61-11ea-88e0-e1cd044c8c5d.png)